java.hints/src/org/netbeans/modules/java/hints/suggestions/ExpandEnhancedForLoop.java

```diff
@@ -96,22 +97,6 @@ public static ErrorDescription run(HintContext ctx) {
         
     }
 
-    private static ExecutableElement findIterable(CompilationInfo info) {
```

Comment:
Why not make this method public, instead of adding new HintUtils class. Please also increment module version in manifest.mf for new api addition.

Reply:
Yes, that makes sense. I will make the necessary changes.